app/experimenter/experiments/api/v5/mutations.py

```diff
@@ -68,8 +68,43 @@ def mutate(cls, root, info, input: ExperimentInput):
         return handle_with_serializer(cls, serializer)
 
 
+class EndExperiment(graphene.Mutation):
```

Comment:
If we expose the `is_end_requested` field in TheOne™️ mutation, we can validate field/status pair in the serializer, and then we don't need a custom mutation at all.  Is that better than having a separate mutation?

Reply:
Yeah, I tried that at one point. Problem is that we have [NimbusStatusRestrictionMixin](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/api/v5/serializers.py#L234-L249) constraining *any* edits via that mutation to just DRAFT status. I started playing with reworking that validator to accept a subset of changes based on the current status, but decided that just adding a new specific mutation was cleaner, at least for now.

Reply:
Ah yep good point 👍